pkg/byteconv/byteconv.go

```diff
@@ -21,9 +22,17 @@ func ParseBool(b []byte) (bool, error) {
 }
 
 func ParseIP(b []byte) (net.IP, error) {
-	ip := net.ParseIP(UnsafeString(b))
+	unsafeStr := UnsafeString(b)
+	ip := net.ParseIP(unsafeStr)
```

Comment:
```suggestion
	ip := net.ParseIP(UnsafeString(b))
```

Reply:
My suggestions are to keep the UnsafeString usage contained to the one spot where its seemingly safe & provides a potential efficiency gain.